spec/grape/validations/validators/allow_blank_spec.rb

```diff
@@ -488,6 +488,11 @@ def app
       get '/disallow_boolean_blank', val: false
       expect(last_response.status).to eq(200)
     end
+
+    it 'accepts value when time allow_blank' do
+      get '/disallow_datetime_blank', val: Time.now
```

Comment:
Here a start of the error stack:
<pre>
Failure/Error: return if value == false || value.present?
     
     NoMethodError:
       undefined method `to_datetime' for false:FalseClass
     # .../.rvm/gems/ruby-2.3.0/gems/activesupport-3.2.19/lib/active_support/core_ext/date_time/calculations.rb:141:in `<=>'
     # ./lib/grape/validations/validators/allow_blank.rb:24:in `=='
     # ./lib/grape/validations/validators/allow_blank.rb:24:in `validate_param!'
</pre>

Here 'active_support/core_ext/date_time/calculations.rb:141':
<pre>
  # Layers additional behavior on DateTime#<=> so that Time and
  # ActiveSupport::TimeWithZone instances can be compared with a DateTime.
  def <=>(other)
    super other.to_datetime
  end
</pre>
I'm not sure why ActiveSupport is calling (if you have any idea, please share, I'll be very grateful). I think the problem is much deeper, but I'm to lazy to digging it up. So, maybe solution what offered is not a solution, it's looks like hook, but it works fine and looks pretty good =)
This works fine too:
<pre>return value.present? || false == value</pre>